source/_components/honeywell.markdown

```diff
@@ -16,54 +16,72 @@ redirect_from:
  - /components/climate.honeywell/
 ---
 
+<p class='note'>
```

Comment:
Please remove this note

Reply:
Please let me know what you think of this version - happy to make more tweaks.

I have asked on discord if someone will cherry-pick the corresponding HA PR to beta.

Reply:
I think @balloob has said I can leave this in. https://github.com/home-assistant/home-assistant.io/pull/9587#discussion_r292573937, so I pushed another commit.

I have taken out the other notes - let me know if this is OK.